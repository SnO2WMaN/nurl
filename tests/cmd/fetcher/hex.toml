args = ["https://hex.pm/packages/phoenix", "1.6.0"]
