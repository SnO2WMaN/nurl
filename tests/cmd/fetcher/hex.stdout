fetchHex {
  pkg = "phoenix";
  version = "1.6.0";
  sha256 = "sha256-Uv/dMfLa6zmbLh61fUaPmaGtbu5djqGdI1NJLwbJ/JY=";
}